### ISSUE TITLE
Allow accessible, disabled menu items in combobox

### ISSUE THREAD

**Issue by davidlygagnon** [salesforce/design-system-react#1635]
Accessibility team and lightning dashboard are requesting this. I'm willing to work on it. 

They currently use the lookup component. Should we create a PR to fix it on the deprecated component so they can be unblocked?

I told them that they should probably upgrade to use the combobox but they might not have time before feature freeze.

@interactivellama What do you think? I can create a PR to have both attributes on both components.

**Comment by interactivellama** [comment]
I would suggest pulling lookup into your project code if you need to ship it for this release and fix it there.

As to `combobox`, I don't see any reference to use of aria-describedby and aria-disabled in these examples? Is the UX pattern in SLDS? https://www.lightningdesignsystem.com/components/combobox/#Entity-Autocomplete-Combobox-(Lookup)

If it's outside SLDS, we can allow it. `aria-describedby` already exists https://react.lightningdesignsystem.com/components/comboboxes/#prop-aria-describedby we can add an `aria-disabled` if that is helpful. I would recommand allowing any `aria-*` props to get passed down to the `input` from Combobox similar to how `data-*` props are handled with this utility: https://github.com/salesforce/design-system-react/blob/master/utilities/get-data-props.js

I don't know what what that does or what UX pattern this fits within. Is this for when the menu doesn't open, but you can tab into the `input`?

**Comment by interactivellama** [comment]
You can do an additional `Combobox` prop if you need to due to time, but this would be the most sustainable preference for the future, so that we don't have partial support for some components. #1636

**Comment by interactivellama** [comment]
## Update
`aria-describedby` and `aria-disabled` are missing from the menu item.

```
            <li role="presentation" class="slds-listbox__item">
              <div id="option3" class="slds-media slds-listbox__option slds-listbox__option_entity slds-listbox__option_has-meta" role="option" aria-disabled="true" aria-describedby="option3help">
                <span class="slds-m-right_xx-small slds-icon slds-icon-text-error slds-icon_x-small">
                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" id="error" width="100%" height="100%"><path d="M12 .9C5.9.9.9 5.9.9 12s5 11.1 11.1 11.1 11.1-5 11.1-11.1S18.1.9 12 .9zM3.7 12c0-4.6 3.7-8.3 8.3-8.3 1.8 0 3.5.5 4.8 1.5L5.2 16.8c-1-1.3-1.5-3-1.5-4.8zm8.3 8.3c-1.8 0-3.5-.5-4.8-1.5L18.8 7.2c1 1.3 1.5 3 1.5 4.8 0 4.6-3.7 8.3-8.3 8.3z"></path></svg>
                </span>
                <span class="slds-media__body">
                  <span class="slds-listbox__option-text slds-listbox__option-text_entity">Sadness</span>
                </span>
              </div>
            </li>
```

and this would point to a `Tooltip`.
